src/core/qgsvectorlayerfeaturecounter.h

```diff
@@ -40,22 +42,45 @@ class CORE_EXPORT QgsVectorLayerFeatureCounter : public QgsTask
      */
     QgsVectorLayerFeatureCounter( QgsVectorLayer *layer, const QgsExpressionContext &context = QgsExpressionContext() );
 
+
+    /**
+     * Calculate the feature count and Ids per symbol
```

Comment:
```suggestion
     * Calculates the feature count and Ids per symbol
```

just for consistency. Or just drop the note altogether, that's better explained on class level (while this method is an implementation detail required by the task api).